apps/src/p5lab/P5LabVisualizationHeader.jsx

```diff
@@ -104,25 +103,23 @@ class P5LabVisualizationHeader extends React.Component {
                     : msg.animationMode()}
                 </button>
               )}
-              {allowAnimationMode &&
-                this.props.isBlockly &&
-                experiments.isEnabled(experiments.BACKGROUNDS_AND_UPLOAD) && (
-                  <button
-                    style={{
-                      ...styles.buttonFocus,
-                      // All truncation if the visualize column is very small
-                      // or if translated strings are longer than English.
-                      overflow: 'hidden',
-                      textOverflow: 'ellipsis',
-                      whiteSpace: 'nowrap',
-                    }}
-                    type="button"
-                    value={P5LabInterfaceMode.BACKGROUND}
-                    id="backgroundMode"
-                  >
-                    {msg.backgroundMode()}
-                  </button>
-                )}
+              {allowAnimationMode && this.props.isBlockly && (
+                <button
+                  style={{
+                    ...styles.buttonFocus,
+                    // All truncation if the visualize column is very small
+                    // or if translated strings are longer than English.
```

Comment:
Obviously not new to this PR, but I don't quite understand this comment.

Reply:
Good find! I added this comment here: https://github.com/code-dot-org/code-dot-org/pull/50220

It's poorly written and meant to say indicate that the style rules allow the text to be truncated if it doesn't fit within the visualization column. See the GIF in the above PR for a visual. Will update!